cg/meta/workflow/microsalt.py

```diff
@@ -138,6 +142,7 @@ def get_organism(self, sample_obj: models.Sample) -> str:
 
         if organism == "VRE":
             reference = sample_obj.organism.reference_genome
+            has_comment = bool(comment)
```

Comment:
Why not just: `elif comment:` on line 150?

Reply:
Do not know. I think I will leave it as to not go to much out of scope